documentation/source/user_section/tutorials/gm-wm/improving-registration-with-gm-seg/gm-informed-mt-template-registration.rst

```diff
@@ -0,0 +1,39 @@
+.. _gm-informed-mt-registration:
+
+Reusing the GM-informed warping field to improve MTI registration
+#################################################################
+
+Now that we have estimated a transformation between the template and the T2* data, we can use this warping field to improve a registration command from :ref:`the previous MT registration tutorial step <mt-registraton-with-anat>`.
+
+First, we return to the ``mt`` directory.
+
+.. code::
+
+   cd ../mt
+
+Next, we run ``sct_register_multimodal`` to compute the transformation between the MT space and the PAM50 template space.
+
+.. code:: sh
+
+   sct_register_multimodal -i "${SCT_DIR}/data/PAM50/template/PAM50_t2.nii.gz" \
+                           -iseg "${SCT_DIR}/data/PAM50/template/PAM50_cord.nii.gz" \
+                           -d mt1.nii.gz \
+                           -dseg mt1_seg.nii.gz \
+                           -param step=1,type=seg,algo=centermass:step=2,type=seg,algo=bsplinesyn,slicewise=1,iter=3 \
+                           -m mask_mt1.nii.gz \
+                           -initwarp ../t2s/warp_template2t2s.nii.gz \
+                           -owarp warp_template2mt.nii.gz \
+                           -qc ~/qc_singleSubj
+
+:Input arguments:
```

Comment:
I get `FileNotFoundError: No such file or no access: 'mt1_seg.nii.gz'`. Was it an output of a previous tutorial? If yes, may be we can provide this file independently because tuts are assumed to be independent. Currently, I only have `mt0.nii.gz  mt1.nii.gz` .

Reply:
* The tutorial these files were generated in was [1.5 Registering Additional Contrasts](https://spinalcordtoolbox--3493.org.readthedocs.build/user_section/tutorials/anat/registering-additional-contrasts.html.)
* The file should also be present in the [`data_improving-registration-with-gm-seg.zip`](https://github.com/spinalcordtoolbox/sct_tutorial_data/releases/download/r20210830/data_improving-registration-with-gm-seg.zip) download (for the individual tutorial).